src/execution_plan/ops/op.c

```diff
@@ -153,6 +155,11 @@ inline void OpBase_DeleteRecord(Record r) {
 	ExecutionPlan_ReturnRecord(r->owner, r);
 }
 
+OpBase *OpBase_Clone(const struct ExecutionPlan *plan, const OpBase *op) {
+	if(op->clone) return op->clone(plan, op);
+	return NULL;
```

Comment:
I expect that we'll ultimately want this function to work with the recursive logic of the Propagate functions, but it's not important right now!